src/main/features/dashboard/index.ts

```diff
@@ -20,6 +27,17 @@ function requestHandler (): express.RequestHandler {
   return AuthorizationMiddleware.requestHandler(requiredRoles, accessDeniedCallback, unprotectedPaths)
 }
 
+function render (claim: Claim, type: string): string {
+  const dashboardName = ClaimStatusFlow.dashboardFor(claim)
+  try {
+    const template = nunjucks.render(path.join(__dirname, './views', 'status', type, dashboardName + '.njk').toString(), { claim: claim })
+    return app.settings.nunjucksEnv.filters['safe'](template)
+  } catch (err) {
+    logger.error(`view not found for status: ${dashboardName}`)
+    return ''
```

Comment:
wondering if there is any "something has gone wrong" kind of status page will be useful here!

@robertparkinson do we have any invalid status page??

Reply:
at the moment production just shows a blank status if we've missed one. same for the case details view. Probably better to show some sort of error to the user

Reply:
yes, that is an option - right now returning `' '` will default to the "old way" of doing things - but if something is not covered in the old way, then it's going to be blank